src/credentials/CHIPCert.h

```diff
@@ -274,21 +267,24 @@ class ChipDN
 };
 
 /**
- *  @struct CertificateKeyId
- *
- *  @brief
- *    A data structure representing a certificate key identifier.
+ *  @brief  A data structure for holding a certificate key identifier, without the ownership of it.
  */
-struct CertificateKeyId
-{
-    const uint8_t * mId = nullptr; /**< Pointer to the key identifier. Encoded as Octet String and represented as the ASN.1 DER
-                                        Integer (X.690 standard). */
-    uint8_t mLen = 0;              /**< Key identifier length. */
+typedef FixedByteSpan<kKeyIdentifierLength> CertificateKeyId;
```

Comment:
nit: Prefer using to typedef

`using CertificateKeyId = FixedByteSpan<kKeyIdentifierLength>;`

and 3 below

Reply:
done